fetch.js

```diff
@@ -8,14 +8,6 @@
   var support = {
     searchParams: 'URLSearchParams' in self,
     iterable: 'Symbol' in self && 'iterator' in Symbol,
-    blob: 'FileReader' in self && 'Blob' in self && (function() {
```

Comment:
I vote that we keep this feature test instead of checking for `self.Blob` presence. It's more comprehensive.